### ISSUE TITLE
Accessibility:List: Tooltip provided for button does not convey meaningful information about the UI. 

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3485]
Description: 

The tooltip “Title1” provided for button does not convey meaningful information about the UI. 
Also, tooltip “Title1” is not read when button is accessed. 
Read as “detail button to activate press spacebar”. 

Expected: Text alternative defining the purpose of UI should be provided for each & every non-text UI element. Also, text alternative/tooltip should be read out when UI is accessed. 

Screenshot:

![image](https://user-images.githubusercontent.com/32538291/94649063-84aa1b00-0311-11eb-9948-b08b3ff8abb4.png)

**Comment by sKudum** [comment]
duplicate of #3486

**Comment by sKudum** [comment]
Opt message is added for voice over.
